### PR TITLE
Remove speed test for map and dict

### PR DESCRIPTION
**Pull request by larrylv**
I run the speed test about 10 times, and every time `Map` runs faster than `HashDict`. I think it's better to remove this speed comparison is incorrect and misleading.

### CODE REVIEW

**Comment by eddwardo** [comment]
Maybe benchmark results changes as the elixir grows?

**Comment by larrylv** [comment]
@eddwardo Yes, so that's why I think we should remove the benchmark comparison.

**Comment by seven1m** [comment]
Need to rename the file and update the readme since there is no Dict code in it any more.

**Comment by larrylv** [comment]
@seven1m Updated, please have a look.

File: 05-map.exs [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# http://elixir-lang.org/docs/stable/elixir/Dict.html
```
Comment (eddwardo) [inline 05-map.exs]: Why is this removed?

Comment (larrylv) [inline 05-map.exs]: Because only `Map` in the file.
